siliconcompiler/core.py

```diff
@@ -508,7 +508,7 @@ def load_target(self, name):
             func(self)
         else:
             self.logger.error(f'Module {name} not found.')
-            sys.exit(1)
+            raise SiliconCompilerError(f'Module {name} not found.')
```

Comment:
Shouldn't this be `Target {name} not found.`

Reply:
I believe that 'Module' is being used in the sense of a [Python module](https://docs.python.org/3/tutorial/modules.html) in these messages.

These methods dynamically search for a function called `setup` in a module with the given name. If we call `load_target(freepdk45_demo)`, the method will look for the setup function in a module called `freepdk45_demo[.py]`. The `load_target`, `load_lib`, etc methods look in different directories (`targets/`, `libs/`, etc), to allow duplicated target/lib/etc names in the future.

But you make a good point that we should make these messages more precisely-worded, thank you for the feedback!